hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/ResidentialEnvironmentTransformations.scala

```diff
@@ -68,47 +68,46 @@ object ResidentialEnvironmentTransformations {
               rawRecord.getOptional("de_country_01_only_dd")
             else getPastResidenceCountry(rawRecord, pastCountryCount, 1L, "de_country_01_dd"),
           dePastResidenceCountry2 =
-            getPastResidenceCountry(rawRecord, pastCountryCount, 2L, "de_country_02_dd"),
+            getPastResidenceCountry(rawRecord, pastCountryCount, 1L, "de_country_02_dd"),
```

Comment:
Line 69 above here also uses `1L`; should that then be 0??

Reply:
No I think that's good.
Here's the original logic from before recent changes:
https://github.com/DataBiosphere/dog-aging-ingest/commit/384e8b0d89aad198a1312fe6ad7f30ffbe337d7b

Was checking for >1 before because the "only" fields indicate only one country, otherwise there should be more than 1. Good catch!